mycli/main.py

```diff
@@ -13,6 +14,12 @@
 from random import choice
 from io import open
 
+# support StringIO for Python 2 and 3
+try:
+    from StringIO import StringIO
```

Comment:
For Python2, we can get better performance if we do `from cStringIO import StringIO`.
https://docs.python.org/2.6/library/stringio.html#module-cStringIO